Standalone/PythonTests/Automated/benchmark_data_aggregator.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python
+"""
+Copyright (c) Contributors to the Open 3D Engine Project. For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+"""
+
+from argparse import ArgumentParser
+import json
+from pathlib import Path
+import time
+import copy
+import subprocess
+import requests
+
+from ly_test_tools.mars.filebeat_client import FilebeatClient
+
+class BenchmarkDataAggregator:
+    def __init__(self, workspace, logger, test_suite):
+        self.build_dir = Path(workspace.paths.build_directory())
+        self.results_dir = Path(workspace.paths.project()) / 'user' / 'Scripts' / 'PerformanceBenchmarks'
+        self.test_suite = test_suite
+        self.logger = logger
+        self.filebeat_client = FilebeatClient(logger)
+
+    def _process_benchmark(self, benchmark_dir):
+        benchmark_metadata = {}
+        frame_stats = { 'count': 0, 'totalTime': 0 }
+        pass_stats = {}
+
+        metadata_file = benchmark_dir / 'benchmark_metadata.json'
```

Comment:
can we use `os.path.joins()` here instead? this would match our common patterns in most of our python3 code (we don't use `/` any more)